docs/aprender-eosio/desarrollo-de-dapps.md

```diff
@@ -1,22 +1,17 @@
 ---
 id: desarrollo-de-dapps
-title: Desarrollo de Daps
+title: Desarrollo de Dapps
 sidebar_label: Desarollo de Dapps
 ---
 
-## Construyendo Aplicaciones Descentralizadas
+## Porqué EOS es construido para Aplicaciones Descentralizadas
```

Comment:
@xavier506 @JeanVegaD maybe we can say "usado" or "fue diseñado"

Reply:
## EOSIO para el desarrollo de aplicaciones descentralizadas (DApps) ?